### PR TITLE
docs: adding the logo in README.md

### PR DESCRIPTION
**Pull request by adnaanbheda**
### Description 

Inclusion of the React icon in README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
+
+
+  
+<p align="center"><img alt="logo" src="https://reactjs.org/favicon.ico"/></p>
+
 # [React](https://reactjs.org/) &middot; [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/facebook/react/blob/master/LICENSE) [![npm version](https://img.shields.io/npm/v/react.svg?style=flat)](https://www.npmjs.com/package/react) [![CircleCI Status](https://circleci.com/gh/facebook/react.svg?style=shield&circle-token=:circle-token)](https://circleci.com/gh/facebook/react) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)
 
 React is a JavaScript library for building user interfaces.
-
```
Comment (MrHeer) [inline README.md:4]: Why delete this line?

Comment (adnaanbheda) [reply README.md]: Yeah, wasn't intentional, updated.

**Comment by tobiasbueschel** [comment]
Hey @adnaanbheda,

This seems to be a duplicate of https://github.com/facebook/react/pull/19446 and https://github.com/facebook/react/pull/19451 which go in the same direction.

**Comment by adnaanbheda** [comment]
@tobiasbueschel Thanks for the update, those PRs using external sources for logos that may not be in control of the React team, this one relies on the official favicon icon, I think that's more appropriate.

**Comment by tobiasbueschel** [comment]
> @tobiasbueschel Thanks for the update, those PRs are using external sources for the logo that may not be in control of the React team, this one relies on the official favicon icon, I think that's more appropriate.

Fully agree with you, external image sources are not great for React's readme. On that note, I think it would be good to use an SVG logo instead of a `favicon` so that it scales better. Otherwise, it might look rather pixelated:

image

This is based on personal preferences, so take it with a grain of salt => maybe the logo is better placed at the right side of the page? Or perhaps we could center the logo, title and badges?
image

Keen to hear what other people think? I do like the simple design of the current readme, but also don't mind the added logo!

Cheers :)

**Comment by adnaanbheda** [comment]
@tobiasbueschel I did try adding an img but taking into account the rest of the markdown syntax, I decided not to proceed with it since it looked like an outlier in comparison to the rest of the doc, others should chime in to decide what the final state of the read me should be! 

Thanks for your input!

**Pull request closed without merging**